server.go

```diff
@@ -127,6 +146,10 @@ type Server struct {
 
 	mu         sync.RWMutex // protects the serviceMap
 	serviceMap map[string]*service
+
+	// Authorize defines authorization strategy of the server
+	// If Authorization function is not provided, no methods would be allowed
+	Authorize func(peer.ID, string, string) bool
```

Comment:
This should be a field set by a `ServerOption` (therefore unexported). See how it works for other server options.

Reply:
Also, things should work as before when no function is set (that is, all methods allowed)